### PR TITLE
[PM-10667] Implement manifest v3 POC for remaining browsers

### PR DESCRIPTION
**Pull request by cagonzalezcs**
## 🎟️ Tracking

https://bitwarden.atlassian.net/browse/PM-10667

## 📔 Objective

Implementing the minimum amount of work required to allow for manifest v3 to work on all browsers.

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

**Review by dani-garcia — APPROVED** [review]

**Review by vgrassia — DISMISSED** [review]

File: apps/browser/src/autofill/deprecated/overlay/pages/list/autofill-overlay-list.deprecated.ts [hunk 1]
```diff
@@ -395,18 +395,29 @@ class AutofillOverlayList extends AutofillOverlayPageElement {
...
-    if (cipher.icon?.icon) {
-      cipherIcon.classList.add("cipher-icon", "bwi", cipher.icon.icon);
+    if (!cipher.icon?.icon) {
+      cipherIcon.append(buildSvgDomElement(globeIcon));
       return cipherIcon;
     }
 
-    cipherIcon.append(buildSvgDomElement(globeIcon));
+    const iconClasses = cipher.icon.icon.split(" ");
+    cipherIcon.classList.add("cipher-icon", "bwi", ...iconClasses);
```
Comment (jprusik) [inline apps/browser/src/autofill/deprecated/overlay/pages/list/autofill-overlay-list.deprecated.ts:420]: non-blocking nit; any reason we need to invert the logic here? It's a little easier to scan in the original logic, imo

Comment (cagonzalezcs) [reply apps/browser/src/autofill/deprecated/overlay/pages/list/autofill-overlay-list.deprecated.ts:420]: I can change it, would this be preferrable?

```
    if (cipher.icon?.icon) {
      const iconClasses = cipher.icon.icon.split(" ");
      cipherIcon.classList.add("cipher-icon", "bwi", ...iconClasses);

      return cipherIcon;
    }

    cipherIcon.append(buildSvgDomElement(globeIcon));
```

Comment (cagonzalezcs) [reply apps/browser/src/autofill/deprecated/overlay/pages/list/autofill-overlay-list.deprecated.ts]: I've gone ahead and incorporated your suggestion.

File: apps/browser/webpack.config.js [hunk 2]
```diff
@@ -13,6 +13,52 @@ if (process.env.NODE_ENV == null) {
...
+  if (browser === "chrome") {
+    // Remove unsupported properties
+    delete manifest.applications;
+    delete manifest.sidebar_action;
+    delete manifest.commands._execute_sidebar_action;
+
+    return JSON.stringify(manifest, null, 2);
+  }
+
+  // // Update the background script reference to be an event page
```
Comment (jprusik) [inline apps/browser/webpack.config.js:34]:
```suggestion
  // Update the background script reference to be an event page
```

**Review by jprusik — DISMISSED** [review]

**Review by jprusik — APPROVED** [review]

**Review by justindbaur — APPROVED** [review]

**Review by vgrassia — APPROVED** [review]

**Pull request merged**